libsel4debug/src/backtrace.c

```diff
@@ -4,12 +4,18 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 #include <autoconf.h>
+#include <sel4/types.h>
+
+/* Don't instrument seL4_GetIPCBuffer so we don't recurse.
```

Comment:
Can you give a few more details where/why the recrusion happens?

Reply:
I updated the comment.